core/protos/protos/network.proto

```diff
@@ -65,6 +65,7 @@ message Message {
     message BlockResponse {
         uint64 request_id = 1;
         repeated CoupledBlock response = 2;
+        uint64 best_index = 3;
```

Comment:
`block_index`? or what is this?

Reply:
Peer responds with their current best index in case they produced more blocks/request before didn't fetch all the blocks.